yarn-project/circuits.js/src/crs/index.ts

```diff
@@ -27,7 +27,7 @@ export class NetCrs {
     const g1End = g1Start + this.numPoints * 64 - 1;
 
     // Download required range of data.
-    const response = await fetch('https://aztec-ignition.s3.amazonaws.com/MAIN%20IGNITION/sealed/transcript00.dat', {
+    const response = await fetch('https://aztec-ignition.s3.amazonaws.com/MAIN%20IGNITION/monomial/transcript00.dat', {
```

Comment:
I don't think this should have been working before, if barretenberg logic now just assumes that you have the generator point already, unless it is being pasted in somewhere else